### PR TITLE
Update GDScript completion names for Pool*Arrays

### LINKED ISSUE #9638: GDScript completion for Pool*Arrays still uses the old names
**Operating system or device - Godot version:**
Windows 10 – Godot 3.0 alpha (9a502d3c17d4701b0fd8390e515c52a69dc0bf85)

**Issue description:**
GDScript completion wasn't updated to the new `Pool*Array` types and still show the old `*Array` format (like `Vector2Array`).

Should be an easy fix.

### PR DESCRIPTION
**Pull request by Xrayez**
Notice: GDScript tokenizer used the old `PoolFloatArray` name.
Renamed `PoolFloatArray` to `PoolRealArray`.

Fixes #9638 

P.S. My first pull request to Godot.

### CODE REVIEW

File: core/variant_parser.cpp [hunk 1]
```diff
@@ -1099,7 +1099,7 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 
 			return OK;
 
-		} else if (id == "PoolFloatArray" || id == "FloatArray") {
+		} else if (id == "PoolRealArray" || id == "RealArray") {
```
Comment (vnen) [inline core/variant_parser.cpp]: This will likely break everything.

Comment (Xrayez) [inline core/variant_parser.cpp]: Yeah, I'm not really sure about this change. Besides, it doesn't really pertain to the issue and deserves another pull request. I have removed the changes to this file.
